addons/l10n_tz/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,320 @@
+                        <field name="expression_ids">
+                            <record id="tax_report_special_relief_value" model="account.report.expression">
+                                <field name="label">value</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">08.base</field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="tax_report_imported_services" model="account.report.line">
+                        <field name="name">Value of imported services</field>
+                        <field name="code">imported_services</field>
+                        <field name="expression_ids">
+                            <record id="tax_report_imported_services_expr_value" model="account.report.expression">
+                                <field name="label">value</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">09.base</field>
+                            </record>
+                            <record id="tax_report_imported_services_expr_rate" model="account.report.expression">
+                                <field name="label">rate</field>
+                                <field name="engine">external</field>
+                                <field name="formula">sum</field>
+                                <field name="subformula">editable</field>
+                            </record>
+                            <record id="tax_report_imported_services_expr_amount" model="account.report.expression">
+                                <field name="label">amount</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">10.tax</field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="tax_report_tz_total_output_tax" model="account.report.line">
+                        <field name="name">Total output (sales) and Tax</field>
+                        <field name="code">tz_total_output_tax</field>
+                        <field name="expression_ids">
+                            <record id="tax_report_tz_total_output_tax_expr_value" model="account.report.expression">
+                                <field name="label">value</field>
+                                <field name="engine">aggregation</field>
+                                <field name="formula">
+                                    taxable_supplies.value +
+                                    capital_good_deferred_vat.value +
+                                    zero_rated_supplies.value +
+                                    exempt_supplies.value +
+                                    special_relief.value +
+                                    imported_services.value
```

Comment:
I wouldn't block for that but it is very often more readable to put the operator on the same line as the (second) operand